src/components/QueryEditor/ServiceQueryEditor.tsx

```diff
@@ -79,6 +90,7 @@ export function ServiceQueryEditor({ query, onChange, datasource, range }: Servi
                   id="accountId"
                   options={accountIdOptions}
                   value={query.accountId ?? ''}
+                  disabled={!query.includeLinkedAccounts && serviceQueryType === ServicesQueryType.listSLOs}
```

Comment:
I think for getServices this dropdown can be empty, so Im keeping the default empty value and only disabling it for listSLOs, cause there it can only be enabled if linked accounts is true.